ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -181,6 +181,31 @@
     #img-frame {
         width: 100%;
     }
+
+    @@media print {
+        .no-print {
```

Comment:
Why not use online-only like the other places?

Reply:
I'm trying to create a separation between the CSS that the puzzles use and the CSS that the site uses, since I know that we've been using online-only for the puzzles.  In fact, I probably should change the CSS I added in _Layout to be no-print as well just to keep with that convention, although the semantics really only matter if other people continue to follow them, which luckily doesn't matter TOO much since it's just naming.

Reply:
Of the three options described in this conversation, I think the one currently in the PR is the worst choice. 😃 I have a slight preference for using the same class everywhere, but your alternative of using one class exclusively for puzzle content and another exclusively for server content works too.